packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -493,4 +496,10 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     private reSubmit(content: any, localOpMetadata: unknown) {
         this.reSubmitCore(content, localOpMetadata);
     }
+
+    private applyStashedOp(content: any): unknown {
+        return this.applyStashedOpCore(content);
```

Comment:
Any reason to have applyStashedOpCore? You can make applyStashedOp abstract instead.